app/scripts/controllers/onboarding.ts

```diff
@@ -0,0 +1,108 @@
+import { ObservableStore } from '@metamask/obs-store';
+import log from 'loglevel';
+
+type firstTimeFlowTypes = 'create' | 'import';
```

Comment:
Can this be an enum instead of a union?

Reply:
It would help because we could use it everywhere these values are currently just strings or using the old constant that is only somewhat used

Reply:
done